cudatoolkit-dev/post-link.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash 
+
+$PREFIX/bin/cuda-install
+cp $PREFIX/bin/cuda-install $CONDA_PREFIX/bin/
```

Comment:
Isn't this called `cuda-install.sh`, not `cuda-install`?

Reply:
Sorry for the confusion! I am realizing that there is unnecessary duplication in some scripts. In `build.sh`, after copying the cuda-install.sh to `$PREFIX/bin/`, I created a soft link, `cuda-install`. Instead of running `cuda-install.sh` and I am  running `cuda-install`. I am going to remove the soft link and just use `cudatoolkit-dev-install.sh` throughout the installation process.